test/extended/workspacetype/workspace.go

```diff
@@ -16,49 +18,91 @@ var _ = g.Describe("[sig-workspace]", func() {
 
-	g.It("Author:pewang-Medium-[Smoke] Multi levels workspaces lifecycle should works well", func() {
-		g.By("# Create a test workspace under user home workspace should become ready to use")
+	g.It("Author:pewang-Medium-[Smoke] Multi levels workspaces lifecycle should work", func() {
+		g.By("# Create a test workspace as parent workspace")
 		k.SetupWorkSpace()
-		myWorkSpace := k.WorkSpace()
+		parentWorkSpace := k.WorkSpace()
 
-		g.By("# Create a child level workspace should become ready to use")
-		k.SetupWorkSpaceWithSpecificPath(myWorkSpace.ServerURL)
-		mySubWorkSpace := k.WorkSpace()
+		g.By("# Create five child workspaces under the parent workspace")
+		childWorkSpaces := []exutil.WorkSpace{}
+		for i := 1; i <= 5; i++ {
+			g.By(fmt.Sprintf("# Creating child workspace no.%v", i))
+			k.SetupWorkSpaceWithSpecificPath(parentWorkSpace.ServerURL)
+			childWorkSpace := k.WorkSpace()
+			childWorkSpaces = append(childWorkSpaces, childWorkSpace)
+		}
 
-		g.By("# From the home workspace could get the test workspace but couldn't get its child level workspace")
-		output, err := k.WithoutNamespace().WithoutKubeconf().WithoutWorkSpaceServer().Run("get").Args("--server="+myWorkSpace.ParentServerURL, "workspace").Output()
-		o.Expect(err).NotTo(o.HaveOccurred())
-		o.Expect(output).Should(o.And(
-			o.ContainSubstring(myWorkSpace.Name),
-			o.ContainSubstring(myWorkSpace.ServerURL),
-		))
-		o.Expect(output).ShouldNot(o.ContainSubstring(mySubWorkSpace.Name))
-
-		g.By("# From the test workspace could get its child workspace")
-		output, err = k.WithoutNamespace().WithoutKubeconf().WithoutWorkSpaceServer().Run("get").Args("--server="+myWorkSpace.ServerURL, "workspace").Output()
-		o.Expect(err).NotTo(o.HaveOccurred())
-		o.Expect(output).Should(o.And(
-			o.ContainSubstring(mySubWorkSpace.Name),
-			o.ContainSubstring(mySubWorkSpace.ServerURL),
-		))
-
-		g.By("# Delete the child workspace and the test workspace should be successful")
-		// Delete the child workspace and check it deleted successfully
-		output, err = k.WithoutNamespace().WithoutKubeconf().WithoutWorkSpaceServer().Run("delete").Args("--server="+myWorkSpace.ServerURL, "workspace", mySubWorkSpace.Name).Output()
+		g.By(`# From the home workspace, I can list only the test "parent workspace" but cannot list the five "child workspaces"`)
+		homeSubWorkSpaces := k.ListWorkSpacesWithSpecificPath(parentWorkSpace.ParentServerURL)
+		// NOTE: use ContainSubstring as a workaround to WorkSpace name inequality bug: https://url.corp.redhat.com/1878a60
+		homeSubWorkSpacesString := strings.Join(homeSubWorkSpaces, ",")
+		o.Expect(homeSubWorkSpacesString).Should(o.ContainSubstring(parentWorkSpace.Name))
+		for _, childWorkSpace := range childWorkSpaces {
+			o.Expect(homeSubWorkSpacesString).ShouldNot(o.ContainSubstring(childWorkSpace.Name))
+		}
+
+		g.By("# From parent workspace, I can list its child workspace")
+		parentSubWorkSpaces := k.ListWorkSpacesWithSpecificPath(parentWorkSpace.ServerURL)
```

Comment:
can we verify all its columns are printed ? Like verify for these columns to be present in output(NAME      TYPE        PHASE   URL)